ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/cmake/tasks/main.yml

```diff
@@ -23,30 +23,30 @@
 
 - name: Download cmake
   get_url:
-    url: https://cmake.org/files/v3.7/cmake-3.7.2.tar.gz
-    dest: /tmp/cmake-3.7.2.tar.gz
+    url: https://cmake.org/files/v3.11/cmake-3.11.4.tar.gz
+    dest: /tmp/cmake-3.11.4.tar.gz
     mode: 0440
```

Comment:
If there is going to be a newer version of cmake installed, should the older version be removed?

Reply:
I'm using the same version for OpenJDK and OpenJFX, so bumping the version (and removing the older) should work.

Reply:
I'm not convinced of this - what versions of OpenJDK were you building?

Reply:
I don't believe OpenJDK has a dependency on cmake; OpenJ9 on the other hand will in the near future.  I think @dnakamura is who is implementing the use of cmake for OMR and OpenJ9.  Devin, should/would 3.11.4 work okay for OMR/OpenJ9?

Reply:
oh yeah. more than good enough. We try to be compatible with 3.4, because thats the stableish version we have on z/os.

Reply:
I think I'm ok with this. @karianna do you have any remaining concerns?

Reply:
Should be OK